<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfowidgetUi</class>
 <widget class="QWidget" name="InfowidgetUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transform</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="info_plainedit"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pb_save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pb_done">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
